book/src/testnet-validator.md

```diff
@@ -0,0 +1,162 @@
+
+### Step 1. Create validator keys
+
+The Ethereum Foundation provides an "Eth2 launch pad" for each active testnet:
+
+- [Medalla launchpad](https://medalla.launchpad.ethereum.org/)
+- [Zinken launchpad](https://zinken.launchpad.ethereum.org/)
+
+Please follow the steps on the appropriate launch pad site to generate
+validator keys and submit deposits. Make sure you select "Lighthouse" as your
+client.
+
+Move to the next step once you have completed the steps on the launch pad,
+including generating keys via the Python CLI and submitting gETH/ETH deposits.
+
+### Step 2. Start an Eth1 client
+
+Since Eth2 relies upon the Eth1 chain for validator on-boarding, all Eth2 validators must have a connection to an Eth1 node.
+
+We provide instructions for using Geth (the Eth1 client that, by chance, we ended up testing with), but you could use any client that implements the JSON RPC via HTTP. A fast-synced node should be sufficient.
+
+#### Installing Geth
+
+If you're using a Mac, follow the instructions [listed here](https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Mac) to install geth. Otherwise [see here](https://github.com/ethereum/go-ethereum/wiki/Installing-Geth).
+
+#### Starting Geth
+
+Once you have geth installed, use this command to start your Eth1 node:
+
+```bash
+ geth --goerli --http
+```
+
+### Step 3. Install Lighthouse
+
+*Note: Lighthouse only supports Windows via WSL.*
+
+Follow the [Lighthouse Installation Instructions](./installation.md) to install
+Lighthouse from one of the available options.
+
+Proceed to the next step once you've successfully installed Lighthouse and view
+its `--version` info.
+
+> Note: Some of the instructions vary when using Docker, ensure you follow the
+> appropriate sections later in this guide.
+
+### Step 4. Import validator keys to Lighthouse
+
+When Lighthouse is installed, follow the [Importing from the Ethereum 2.0 Launch
+pad](./validator-import-launchpad.md) instructions so the validator client can
+perform your validator duties.
+
+Proceed to the next step once you've successfully imported all validators.
+
+### Step 5. Start Lighthouse
+
+For staking, one needs to run two Lighthouse processes:
+
+- `lighthouse bn`: the "beacon node" which connects to the P2P network and
+	verifies blocks.
+- `lighthouse vc`: the "validator client" which manages validators, using data
+	obtained from the beacon node via a HTTP API.
+
+Starting these processes is different for binary and docker users:
+
+#### Binary users
+
+Those using the pre- or custom-built binaries can start the two processes with:
+
+```bash
+lighthouse --testnet MY_TESTNET bn --staking
+```
+
+```bash
+lighthouse --testnet MY_TESTNET vc
+```
+
+#### Docker users
+
+Those using Docker images can start the processes with:
+
+```bash
+$ docker run \
+	--network host \
+	-v $HOME/.lighthouse:/root/.lighthouse sigp/lighthouse \
+	lighthouse --testnet MY_TESTNET beacon --staking --http-address 0.0.0.0
```

Comment:
I've heard users express confusion about when we are using aliases in our docs, so maybe `bn` here for consistency (although maybe we should update it in the docker specific page too)